recipes/scine-xtb/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "xtb" %}
+{% set pkg_name = "scine-" + name %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ pkg_name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/qcscine/xtb_wrapper/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 1c59ec80ed9424b1c81ab1fb2d9a137a6bd5309b00e0e2e1ed58732b5ba45af1
+    fn: {{ name }}.tar.gz
+    patches:
+      - pkgconfig.patch
+  - url: https://github.com/qcscine/development-utils/archive/refs/tags/4.0.0.tar.gz
+    sha256: 54002c2082b6bb75672ec66bf9cf3935bbdf6b085ed9b4d7174cbdedb7c2275d
+    fn: dev.tar.gz
+    folder: dev
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  script:
+    - set -ex
+    - >
+       cmake
+       ${CMAKE_ARGS}
+       -B _build -G Ninja
+       -DSCINE_MARCH=""
+    - cmake --build _build
+    - cmake --install _build
```

Comment:
Can you use a `build.sh` file? Makes it easier for the bot to fix things in the future.